[2024-07-08T03:55:01.137+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.task_validate_submissions scheduled__2024-07-08T03:50:00+00:00 [queued]>
[2024-07-08T03:55:01.145+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.task_validate_submissions scheduled__2024-07-08T03:50:00+00:00 [queued]>
[2024-07-08T03:55:01.145+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-07-08T03:55:01.156+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): task_validate_submissions> on 2024-07-08 03:50:00+00:00
[2024-07-08T03:55:01.162+0000] {standard_task_runner.py:57} INFO - Started process 1117 to run task
[2024-07-08T03:55:01.169+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'task_validate_submissions', 'scheduled__2024-07-08T03:50:00+00:00', '--job-id', '3472', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp2xmkqom2']
[2024-07-08T03:55:01.173+0000] {standard_task_runner.py:85} INFO - Job 3472: Subtask task_validate_submissions
[2024-07-08T03:55:01.262+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.task_validate_submissions scheduled__2024-07-08T03:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-08T03:55:01.376+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='task_validate_submissions' AIRFLOW_CTX_EXECUTION_DATE='2024-07-08T03:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-08T03:50:00+00:00'
[2024-07-08T03:55:01.378+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-08T03:55:01.393+0000] {logging_mixin.py:151} INFO - 2024-01-31 00:00:00
[2024-07-08T03:55:01.393+0000] {logging_mixin.py:151} INFO - {'name': '1HK', 'version': 'v0', 'bic4': '0000', 'date': '2024-01-31 00:00:00'}
[2024-07-08T03:55:01.394+0000] {logging_mixin.py:151} INFO - PostgreSQL connection is closed
[2024-07-08T03:55:01.394+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-07-08T03:55:01.404+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=task_validate_submissions, execution_date=20240708T035000, start_date=20240708T035501, end_date=20240708T035501
[2024-07-08T03:55:01.457+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-08T03:55:01.487+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
